app.py

```
@@ -74,7 +89,10 @@ def show_image(self):
             # Add buttons to the popup window
             button_frame = tk.Frame(popup)
             button_frame.pack(pady=10)
-
+        except Exception as e:
```

Ruff

Do not catch blind exception: `Exception`

(BLE001)